context-py/src/lib.rs

```diff
@@ -123,8 +123,12 @@ pub fn parse_meta_from_tarball(
 
 #[gen_stub_pyfunction]
 #[pyfunction]
-fn codeowners_parse(codeowners_bytes: Vec<u8>) -> CodeOwners {
-    CodeOwners::parse(codeowners_bytes)
+fn codeowners_parse(codeowners_bytes: Vec<u8>) -> PyResult<BindingsOwners> {
+    let codeowners = CodeOwners::parse(codeowners_bytes);
```

Comment:
Now that `CodeOwners` isn't directly given to Python, we can remove the `pyclass` macro from it

Reply:
I think you need to use `m.add_class` for `BindingsOwners` below too

Reply:
Turns out that breaks bundle_meta for some reason, so I'm leaving it alone for now.

Reply:
That's because we're still registering it to Python below:
https://github.com/trunk-io/analytics-cli/pull/195/files#r1872087986